<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>WeReWolf</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPlay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato Semibold</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="textServerPort">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>55555</string>
    </property>
    <property name="cursorPosition">
     <number>5</number>
    </property>
    <property name="placeholderText">
     <string>55555</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNickname">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nickname</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelServerIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textServerIP">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>localhost</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelServerPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Server Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textNickname">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>user_name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Client Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textClientPort">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>9999</string>
    </property>
    <property name="placeholderText">
     <string>9999</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
